src/resolvers-cassandra/Messages/queries.js

```diff
@@ -54,18 +58,15 @@ function queryEventsTable(rowsWithEventIds, args) {
       eventsParams.push(`%${args.fulltextTerm}%`);
     }
 
-    const limit = parseLimit(args.limit);
-    if (limit) {
-      eventsQuery += ' LIMIT ?';
-      eventsParams.push(limit);
-    }
-
     cassandraConnector.executeQuery(eventsQuery, eventsParams)
     .then(rows => {
       const features = rows.map(eventToFeature);
+      const pageState = eventIdResponse.pageState;
 
       resolve({
-        features
+        type: 'FeatureCollection',
+        features, pageState,
```

Comment:
Nit: One property per line is easier to read.

```
resolve({
  type: 'FeatureCollection',
  features,
  pageState,
  bbox: args.bbox
})
```

Nit: it seems a bit arbitrary why some of the properties are defined inline and some are defined as const variables above. Elsewhere in the project this is solved by just always declaring the variables that get resolved as const, so:

```js
const features = rows.map(eventToFeature);
const pageState = eventIdResponse.pageState;
const bbox = args.bbox;
const type = 'FeatureCollection';

resolve({
  type,
  features,
  pageState,
  bbox
});
```

Don't particularly care either way, just wanted to point out the inconsistency.